test/Npgsql.Tests/TestUtil.cs

```diff
@@ -159,7 +159,7 @@ public static async Task<object> ExecuteScalarAsync(this NpgsqlConnection conn,
 
     public static class NpgsqlCommandExtensions
     {
-        public static T ExecuteScalar<T>(this NpgsqlCommand cmd)
+        public static T ExecuteScalar<T>(this NpgsqlCommand cmd) where T : struct
```

Comment:
Accidental? Doesn't matter, this is experimental in any case :)

Reply:
After upgrading to netcoreapp5.0 the comiler started giving me some errors about nullable references. This method was used only for decimals, so the fix was to restrict `T` to `structs`